src/domain/Lotto.js

```diff
@@ -0,0 +1,78 @@
+import generateRandomNumberInRange from '../util/generateRandomNumberInRange';
+import LOTTO_RULE from '../constants/rules/lottoRule';
+import ERROR_MESSAGE from '../constants/messages/errorMessage';
+
+export default class Lotto {
+  #lottoNumbers = [];
+
+  constructor(lottoNumbers = []) {
+    const needRandomGenerating = lottoNumbers.length === 0 ? true : false;
+
+    needRandomGenerating ? this.#drawAutoLottoNumbers() : this.#makeCustomLottoNumbers(lottoNumbers);
+    this.#validateLotto();
+    this.#sortLottoNumbers();
+  }
+
+  //TODO: validate 파일 분리?
+  #validateLotto() {
+    this.#isNotValidLottoNumberCount();
+    this.#hasRedundentLottoNumber();
+
+    this.#lottoNumbers.forEach(lottoNumber => {
+      this.#isNotNumber(lottoNumber);
+      this.#isInvalidLottoNumberRange(lottoNumber);
+    });
+  }
+
+  #isNotValidLottoNumberCount() {
```

Comment:
그런데 요런 validation 함수에 `isNot` 과 같이 부정표현을 써주신 이유가 있나요?
요런건 긍정문으로 바뀌면 더 잘읽히기도 하고 의미에도 더 맞을 것 같아요
```suggestion
  #isValidLottoNumberCount() {
```

Reply:
특별한 이유 없이.. 그냥 떠오르는 것으로 부정표현을 사용하게 되었습니다. 
긍정표현으로 변경하겠습니다! 좋은 피드백 감사합니다 :)

Reply:
네네! ㅎㅎ 
아무래도 부정표현은 사람이 두번 생각을 해야하니까 가독성을 해치게 되는 것 같더라고요!